### PR TITLE
Modernize the build tools

### PR DESCRIPTION
**Pull request by mweibel**

### CODE REVIEW

File: src/dto.js [hunk 1]
```diff
@@ -21,52 +20,52 @@ class BaseDTO {
...
    */
-  toJSON() {
-    const values = {};
+  toJSON () {
+    const values = {}
 
     Object.keys(this)
-      .filter(function(key) {
-        return key !== '__RAW__';
+      .filter(function (key) {
```
Comment (pstadler) [inline src/dto.js:61]: maybe use arrow functions across the board?

File: src/dto.js [hunk 2]
```diff
@@ -141,13 +140,13 @@ var methods = [
...
  * @returns {*}
  */
-methods.forEach(function(method) {
-  BaseListDTO.prototype[method] = function() {
-    return [][method].apply(this.items || [], arguments);
-  };
-});
+methods.forEach(function (method) {
+  BaseListDTO.prototype[method] = function () {
+    return [][method].apply(this.items || [], arguments)
```
Comment (pstadler) [inline src/dto.js:145]: could we modernize this maybe? `apply` may not be required any more using spread and stuff

File: src/errors.js [hunk 3]
```diff
@@ -15,69 +15,44 @@ const inherits = require('inherits');
  *
  * @param {string} msg error message
  */
-function BaseError(msg) {
-  BaseError.super_.call(this, msg);
-  if(Error.captureStackTrace) {
-    Error.captureStackTrace(this, this.constructor);
+function BaseError (msg) {
+  BaseError.super_.call(this, msg)
```
Comment (pstadler) [inline src/errors.js:19]: `super(msg)` should do the trick

File: src/dto.js (hunk 1 above)
Comment (mweibel) [reply src/dto.js:61]: we can, but I'd prefer a separate PR for that

File: src/dto.js (hunk 2 above)
Comment (mweibel) [reply src/dto.js:145]: we can, but I'd prefer a separate PR for that

File: src/errors.js (hunk 3 above)
Comment (mweibel) [reply src/errors.js:19]: we can, but I'd prefer a separate PR for that

**Review by pstadler — APPROVED** [review]

**Pull request merged**